### ISSUE TITLE
New get_tlm_cnt functionality

### ISSUE THREAD

**Issue by sunws4me** [OpenC3/cosmos#1534]
current implementation takes a target and packet.  a version that also took a mnemonic and use LATEST for the packet would be great.

**Comment by jmthomas** [comment]
The api is to request how many times a given packet has been received. LATEST doesn't really make sense here as it means the last item from any packet. So you'd never really know what count you were getting.

**Comment by jmthomas** [comment]
Makes sense ... we have the ability to check for items within packets so I think this is possible
